bip-musig2.mediawiki

```diff
@@ -55,13 +55,22 @@ The MuSig2 variant in this specification stands out by combining all of the foll
 * '''MuSig2* optimization''': The specification uses an optimization that allows saving a point multiplication in key aggregation. The MuSig2 scheme with this optimization is called MuSig2* and proven secure in the appendix of the [https://eprint.iacr.org/2020/1261 MuSig2 paper]. The optimization is that the second distinct key in the list of public keys given to the key aggregation algorithm (as well as any keys identical to this key) gets the constant key aggregation coefficient ''1''.
 * '''Parameterization of MuSig2 and security''': In this specification, each signer's nonce consists of two elliptic curve points. The [https://eprint.iacr.org/2020/1261 MuSig2 paper] gives distinct security proofs depending on the number of points that constitute a nonce. See section [[#choosing-the-size-of-the-nonce|Choosing the Size of the Nonce]] for a discussion.
 
+== Overview ==
+
+Implementors of the specification must make sure to understand this section thoroughly to avoid subtle mistakes that may lead to catastrophic failure.
```

Comment:
micro nit:
```suggestion
Implementers of the specification must make sure to understand this section thoroughly to avoid subtle mistakes that may lead to catastrophic failure.
```
(seems more common https://english.stackexchange.com/questions/195699/implementor-vs-implementer )

Reply:
also changed this at another location